shell/app-shell/elements/arc-planner.js

```diff
@@ -61,11 +61,15 @@ class ArcPlanner extends Xen.Debug(Xen.Base, log) {
       this._instantiateSuggestion(arc, pendingSuggestions.shift());
     }
   }
+
   async _createPlanificator(arc, userid) {
-    const protocol = ArcsUtils.getUrlParam('planificatorProtocol') || 'volatile';
+    const storageKeyBase = ArcsUtils.getUrlParam('plannerStorageKeyBase')
+        || ArcsUtils.getUrlParam('storageKeyBase')
```

Comment:
why do we need both - storageKeyBase and plannerStorageKeyBase?

Reply:
plannerStorageKeyBase defaults to storageKeyBase, if set.  That allows you to set both with one query param.